src/iiif-collection.json

```
@@ -904,6 +904,24 @@
       "label": "AV",
       "visible": true,
       "manifests": [
+        {
+          "@id": "https://iiif.io/api/cookbook/recipe/0002-mvm-audio/manifest.json",
+          "@type": "sc:Manifest",
+          "label": "Simplest Audio Example 1",
+          "visible": true
+        },
+        {
+          "@id": "https://iiif.io/api/cookbook/recipe/0003-mvm-video/manifest.json",
+          "@type": "sc:Manifest",
+          "label": "Simplest Video Example 3",
```

We jump from example 1 to example 3? Should this be example 2, or is there a reason for this numbering?

```suggestion
          "label": "Simplest Video Example 2",
```

---

it's from the cookbook: https://iiif.io/api/cookbook/recipe/0003-mvm-video/